scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Sugar.scala

```diff
@@ -0,0 +1,13 @@
+package scala.meta
+package semantic
+
+import org.scalameta.data._
+import scala.meta.inputs._
+
+@data class Sugar(
+    input: Input.Sugar,
+    names: List[(Position, Symbol)]
```

Comment:
Whoops, this never became a Map, I changed `Attributes.names` into a Map. Will update Sugar accordingly.